models/repo/repo.go

```diff
@@ -646,14 +646,16 @@ func (repo *Repository) DescriptionHTML(ctx context.Context) template.HTML {
 type CloneLink struct {
 	SSH   string
 	HTTPS string
+	Tea   string
 }
 
-// ComposeHTTPSCloneURL returns HTTPS clone URL based on given owner and repository name.
-func ComposeHTTPSCloneURL(ctx context.Context, owner, repo string) string {
+// ComposeHTTPSCloneLink returns HTTPS clone URL based on the given owner and repository name.
+func ComposeHTTPSCloneLink(ctx context.Context, owner, repo string) string {
```

Comment:
Please keep the old names.

In Gitea, usually URL means absolute URL, link means relative link.